addons/google_calendar/models/calendar.py

```diff
@@ -32,6 +35,13 @@ def _compute_google_id(self):
             elif not event.google_id:
                 event.google_id = False
 
+    @api.depends('videocall_location')
+    def _compute_videocall_source(self):
+        events_with_google_url = self.filtered(lambda event:
+            event.videocall_location and self.MEET_ROUTE in event.videocall_location)
```

Comment:
Maybe could be simplified a bit like
```suggestion
        events_with_google_url = self.filtered(lambda event: self.MEET_ROUTE in (event.videocall_location or ''))
```